.circleci/config.yml

```diff
@@ -92,6 +92,11 @@ any-upstream: &any-upstream
     ignore:
       - /pull\/.*/
 
+any-fork: &any-fork
+  branches:
+    only:
+      - /pull\/.*/
+
```

Comment:
This doesn't seem right; seems like it'd work on any pull request.  I _suspect_ the exclusion of  pull/* in any-upstream is to avoid running on the branches which are automatically created when a PR is opened.

Reply:
It doesn't feel right, but there's nothing document in CircleCI's docs for this.
As is right now, it seems only PR's from forks have that branch name.

Reply:
I guess that makes sense.  The PR branch would only need auto-created for pulls where the branch isn't already present in the repository.  I still don't completely like it, since it's not definitely documented anywhere obvious, but it's easy enough to change if we experience problems. :)

As long as we don't hire Any engineers named Patty Ull or similar, we should be ok.